editor/editor_inspector.cpp

```diff
@@ -661,9 +661,10 @@ void EditorProperty::gui_input(const Ref<InputEvent> &p_event) {
 
 		if (keying_rect.has_point(mpos)) {
 			accept_event();
-			emit_signal(SNAME("property_keyed"), property, use_keying_next());
+			const bool l_keying_next = use_keying_next();
```

Comment:
What is "l"? Last? I suggest naming the variables which can be understood by them.

Reply:
Usually we don't use a specific prefix for local variables, they're usually used closed to their definition and so the context is fairly explicit. The conventions we do have is `p_` prefix for const method parameters, and `r_` for method parameters we write data to.

Reply:
We also don't use `const` with primitive types.

Reply:
@Calinou tends to use it for locals... I don't mind either way personally, I think it's unlikely to make a difference so I tend to only use when I actually declare something which I explicitly want to be a local constant (like `const int tile_size = 64;`).

For method parameters on the other it's true that we usually ask not to add `const` for primitive types.

Reply:
As I wrote in [#70058 ](https://github.com/godotengine/godot/issues/70058#issuecomment-1353717592), this fix does not seem to completely solve the problem. Also, this use_keying_next() does not seem to have anything to do with the crash. (But there is a problem with the wrong bindings in the .NET build, so I will send a separate PR for this.)

I'm working on AnimationEditor now, so I'll take a closer look at this issue later.

Reply:
@TokageItLab That's one of the reasons why I mentioned it might not be a terrible idea to add a DEV_ASSERT to the message related to the signal without the deferred connection -- Seems like the real issue here is a pointer being set to null in the inspector without properly being cleaned up.  A deferred connection in this case still arguably makes sense, since it will reduce the likelyhood of invalid pointer access during signal calls. 

`use_keying_next` being assigned to a variable is ultimately unimportant to this bug, so I could just revert that part to remove the distraction. It's simply being stored in the variable to avoid doing duplicate work.

I can look into the source of the null pointer assignment a little later to try to solve the problem you've mentioned in #70058 .

Reply:
The deferred connection avoids crashes, but it is odd that the inspector is about to be changed while keying in the first place. If we remove the behavior of the inspector being changed during keying, the deferred connection should not be necessary.

Reply:
@TokageItLab I see you made a PR that fixes this without the need for this change. Feel free to close this issue and merge the other when you feel the time is right.